graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -1091,7 +1094,11 @@ def _compile_output_step(query_metadata_table):
 ##############
 
 
-def ast_to_ir(schema, ast, type_equivalence_hints=None):
+def ast_to_ir(
+    schema: GraphQLSchema,
+    ast: DocumentNode,
+    type_equivalence_hints: Optional[TypeEquivalenceHintsType] = None,
+) -> IrAndMetadata:
     """Convert the given GraphQL AST object into compiler IR, using the given schema object.
 
     Args:
```

Comment:
We can drop the types in the docstring since we have type hints now (same for `graphql_to_ir` below)

Reply:
Thanks, good call! Will fix shortly.